mithril-client/src/services/snapshot.rs

```diff
@@ -472,9 +473,10 @@ mod tests {
         http_client
             .expect_get_content()
             .returning(|_| {
-                Err(AggregatorHTTPClientError::RemoteServerUnreachable(
-                    "whatever".to_string(),
-                ))
+                Err(AggregatorHTTPClientError::SubsystemError {
+                    message: String::new(),
```

Comment:
Message attribute can be removed since the anyhow error can handle context on its own.